Formula/websocat.rb

```diff
@@ -0,0 +1,24 @@
+class Websocat < Formula
+  desc "Command-line client for WebSockets"
+  homepage "https://github.com/vi/websocat"
+  url "https://github.com/vi/websocat/archive/v1.2.0.tar.gz"
+  sha256 "e1a6021770c9ae5b364eaa74b7435f259c5d84bf5c7210cab7f0c775fa2158a5"
+
+  depends_on "pkg-config" => :build
```

Comment:
You don't need `pkg-config`.
(In `ripgrep` it is required to work with `pcre2`)

Reply:
I didn't remove it because of possible dependency on OpenSSL, which typically relies on pkg-config (at least on Linux).

Shall it relly be removed (or made effective only when using LinuxBrew)?

Reply:
Oh, Linuxbrew was totally outside my consideration.
I agree that `websocat` requires `pkg-config` for `openssl` on Linux.